libraries/AP_Notify/OreoLED_PX4.cpp

```diff
@@ -59,6 +59,7 @@ extern "C" int oreoled_main(int, char **);
 // init - initialised the device
 bool OreoLED_PX4::init()
 {
+/* This code is breaking master when the OreoLeds are enabled in PX4-V2 so I guesss no one uses it and its not needed?
```

Comment:
I can't delete this code as it is needed for the OreoLEDs to work, but its breaking the px4_v2 compile so I need a little feedback from Tridge or Randy as to how to handle this.